recipe/meta.yaml

```diff
@@ -35,7 +34,7 @@ outputs:
         - python >=3.7
         - pyyaml >=5.1
         - toml
-        - virtualenv >=20.0.8
```

Comment:
This should still be 20.0.8. See: https://github.com/pre-commit/pre-commit/blob/v2.20.0/setup.cfg#L31

Reply:
Ah, I think I looked at the main branch. Thank you!